pom.xml

```diff
@@ -3,12 +3,6 @@
   xmlns="http://maven.apache.org/POM/4.0.0"
   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
   <modelVersion>4.0.0</modelVersion>
-  <parent>
-    <groupId>io.confluent</groupId>
-    <artifactId>common-parent</artifactId>
-    <version>7.7.0</version>
-  </parent>
```

Comment:
Answered in PR description:

> It turns out that we were running the integration test suite against the native executable -- not once, but TWICE! Apparently, it was due to the [<id>](https://stackoverflow.com/a/72302045) being set (WTF?) in the failsafe plugin config. I've removed that. **However, that wasn't enough since [confluent/common-parent's pom is also riddled with this oddity](https://github.com/confluentinc/common/blob/v7.7.0/pom.xml#L839). I've removed that as well so ide-sidecar no longer has confluentinc/common-parent as a parent POM.***